on-off-board/onboarding.md

```diff
@@ -18,36 +16,37 @@ This document is intended to help developers become familiar with NYPL digital p
 ### Commonly used accounts
 
 A developer will generally need accounts for the following services:
-- [npm organization](https://www.npmjs.com/org/nypl): We pay by user, consider waiting until someone _needs_ to be added.
-- [Travis CI](https://travis-ci.com): Accounts are OAuthed and synced through Github. Adding/Removing access to Github controls access to Travis.
 - Slack
     - [nypl.slack.com](https://nypl.slack.com/)
-- Github
+- Github (Can use a personal account or make a new NYPL specific github account)
     - Github organizations:
         - [NYPL](https://github.com/NYPL)
         - [NYPL-discovery](https://github.com/NYPL-discovery)
-        - [NYPL-registry](https://github.com/NYPL-registry)
         - [NYPL-Simplified](https://github.com/NYPL-Simplified)
+    - Old and no-longer used organizations
+        - [NYPL-registry](https://github.com/NYPL-registry)
         - [nypl-spacetime](https://github.com/nypl-spacetime)
-        - [nypl-open-audio](https://github.com/nypl-openaudio) (this hasn't been used in a while)
-- [Docker Hub](https://hub.docker.com/u/nypl/)
+        - [nypl-open-audio](https://github.com/nypl-openaudio)
+        - [nypl-publicdomain](https://github.com/NYPL-publicdomain)
 - JIRA*
     - [jira.nypl.org](https://jira.nypl.org/)
-- Bitbucket
-    - [bitbucket.org/NYPL](https://bitbucket.org/NYPL)
+- [Docker Hub](https://hub.docker.com/u/nypl/)
 - Amazon Web Services (AWS)
     - Primary AWS accounts (MFA Login):
         - [nypl-digital-dev](https://nypl-digital-dev.signin.aws.amazon.com/console)
-        - [nypl-sandbox](https://nypl-sandbox.signin.aws.amazon.com/console)
+- Bamboo*: Deployment happens via Bamboo 
+    - [http://bamboo.nypl.org/](http://bamboo.nypl.org/)
 
 ### Less commonly used accounts
 
 - NYPL Platform
     - [https://platform.nypl.org](https://platform.nypl.org)
+- [npm organization](https://www.npmjs.com/org/nypl): You do not need an npm account to publish to npm 
+- [Travis CI](https://travis-ci.com): Accounts are OAuthed and synced through Github. Adding/Removing access to Github controls access to Travis.
+- Bitbucket: Older apps are on bitbucket
+    - [bitbucket.org/NYPL](https://bitbucket.org/NYPL)
```

Comment:
There is also stash.nypl.org which is a private bitbucket repo that some IT/devops stuff lives in, but not sure if that should get a mention here